db/migrate/20191014080804_create_gists.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGists < ActiveRecord::Migration[6.0]
+  def change
+    create_table :gists do |t|
+      t.string :gist_url, null: false
```

Comment:
```suggestion
      t.string :url, null: false
```
зачем дубировать? `gist.gist_url` ?